query/src/main/java/org/infinispan/query/backend/ClassLoaderServiceImpl.java

```diff
@@ -1,78 +0,0 @@
-package org.infinispan.query.backend;
-
-import java.io.InputStream;
-import java.net.URL;
-import java.util.ServiceLoader;
-
-import org.hibernate.search.engine.service.classloading.spi.ClassLoaderService;
-import org.hibernate.search.engine.service.classloading.spi.ClassLoadingException;
-
-/**
- * An implementation of the {@link ClassLoaderService} contract that just delegates loading to an actual {@link
- * ClassLoader}.
- *
- * @author anistor@redhat.com
- * @since 9.2
- */
-final class ClassLoaderServiceImpl implements ClassLoaderService {
```

Comment:
From my understanding ISPN relies on an aggregated class loader as well, I suppose we have here the same issue with WF modules, see `LifecycleManager#makeAggregatedClassLoader`. Anyway I restored the class and use it so that ISPN aggreagated class loader is used. I didn't know we have an SPI to set it, thank you.